script.js

```diff
@@ -0,0 +1,86 @@
+// variables
+const divide = document.getElementById('/')
+const multiply = document.getElementById('*')
+const minus = document.getElementById('-')
+const plus = document.getElementById('+')
+const equals = document.getElementById('=')
+const reset = document.getElementById('reset')
+const numberButtons = document.querySelectorAll('.number-button')
+const displayEl = document.getElementById('calc-display')
+
+// state
+let val1 = 0;
+let val2 = 0;
+let currentOperation = null;
+
+document.addEventListener("DOMContentLoaded", function () {
+    // make sure buttons have values based on thier id
+    function setNumberButtonValues() {
+        numberButtons.forEach(button => {
+            const numericValue = button.id;
+            button.value = numericValue;
+        });
+    }
+
+    // add value to display element
+    function addValueToDisplay(buttonValue) {
+        displayEl.innerText += buttonValue;
+    }
+
+    // reset display screen
+    function resetDisplay() {
+        displayEl.innerText = ''
+    }
+
+    // perform operetion based on /,*,+, or - after we already stored val1
+    function performOperation() {
+        val2 = parseFloat(displayEl.innerText);
+        switch (currentOperation) {
+            case '/':
+                if (val2 !== 0) {
+                    displayEl.innerText = (val1 / val2).toString();
+                } else {
+                    displayEl.innerText = 'No way, Jose';
+                }
+                break;
+            case '*':
+                displayEl.innerText = (val1 * val2).toString();
```

Comment:
no need to convert to string as the DOM when setting innerText will actually do the conversion for you